### ISSUE TITLE
[Request] Allow setting / limiting of rendering framerate.

### ISSUE THREAD

**Issue by hazagames** [KTXSoftware/Kha#320]
Hey guys,

I think allowing setting / limiting of rendering framerate without affecting Schedule is an important feature to have, especially on mobile HTML5. It's not always possible for our games to run at that sexy FPS, a lower but stable framerate may be more preferable at times.

In HTML5 target, messing with `requestAnimationFrame` will affect the timing of Scheduler while sometime we need to keep the update rate and only change the render rate on different platforms / targets. I also tried to render my backbuffer at a lower rate but Kha clears its rendering context every frame, results in flickering. Currently I achieved what I want by messing with `SystemImpl.loadFinished()` and cap the execution of the part below `Scheduler.executeFrame();` in `animate()`. I keep dreaming about a day we can achieve the same thing in a proper way, like setting a frameCap property or something, at least for HTML5 target.

Or is there already a way to do that? Please tell me!

Thank you!

**Comment by RobDangerous** [comment]
Messing with requestAnimationFrame should not affect the Scheduler, so I will fix that first...
A frame cap also sounds like a good idea.

**Comment by TheSHEEEP** [comment]
Agreed. Limiting the framerate is a necessity, graphic cards have been burned to crisp already by the lack of it (StarCraft 2 is probably the most known example).
